pytorch_lightning/metrics/__init__.py

```diff
@@ -38,3 +40,8 @@
     SSIM,
 )
 from pytorch_lightning.metrics.retrieval import RetrievalMAP  # noqa: F401
+
+warn(
+    "`pytorch_lightning.metrics.*` module has been renamed to `torchmetrics.*` and split off to its own package"
+    " (https://github.com/PyTorchLightning/metrics) since v1.3 and will be removed in v1.5", DeprecationWarning
+)
```

Comment:
@Borda with the fixes in #8002 this warning appears every time pl is imported. Should avoid that

Reply:
Do you mean imported in the same running session?
we can have an internal counter to make sure it is raised only once...

Reply:
I mean the following:

```python
$ python -c 'import pytorch_lightning'
/Users/carlosmocholi/git/pytorch-lightning/pytorch_lightning/metrics/__init__.py:43: LightningDeprecationWarning: `pytorch_lightning.metrics.*` module has been renamed to `torchmetrics.*` and split off to its own package (https://github.com/PyTorchLightning/metrics) since v1.3 and will be removed in v1.5
  rank_zero_deprecation(
```

Reply:
I'm getting major log spam with this warning on windows and it appears there is no way to turn it off.

```
import warnings
# from pytorch_lightning.utilities.warnings import LightningDeprecationWarning
# Even specifying LightningDeprecationWarning below does not work
warnings.filterwarnings("ignore", category=DeprecationWarning) 

import pytorch_lightning
```

The above code causes the deprecation warning to print, so when I have a data loader with multiple workers, this message spams the terminal.

Reply:
Yeah, we should remove it as it's too intrusive. Would you like to open a PR?